plenum/test/view_change/test_same_quorum_on_different_nodes_in_promote.py

```diff
@@ -0,0 +1,37 @@
+import pytest as pytest
+
+from plenum.common.config_helper import PNodeConfigHelper
+from plenum.common.constants import STEWARD_STRING, INSTANCE_CHANGE
+
+from plenum.test.delayers import cDelay, ppDelay, pDelay, nv_delay, icDelay
+from plenum.test.helper import waitForViewChange, checkViewNoForNodes
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.node_request.helper import sdk_ensure_pool_functional
+from plenum.test.pool_transactions.helper import sdk_add_new_steward_and_node, sdk_add_new_nym, prepare_new_node_data, \
+    prepare_node_request, sdk_sign_and_send_prepared_request, create_and_start_new_node
+from plenum.test.stasher import delay_rules_without_processing, delay_rules
+from plenum.test.test_node import checkNodesConnected, TestNode, ensureElectionsDone
+from plenum.test.view_change.helper import ensure_view_change_complete
+
+
+nodeCount = 4
+
+
+@pytest.mark.skip(reason="INDY-2322 A lagging node may be the only one who started view change "
```

Comment:
FYI a similar test is already added in https://github.com/hyperledger/indy-plenum/pull/1456